### ISSUE TITLE
Add Cognito integration to STAC Browser

### ISSUE THREAD

**Issue by rtapella** [unity-sds/unity-project-management#149]
stub issue for discussion
Who has bandwidth to integrate STAC Browser with Cognito?

**Comment by rtapella** [comment]
U-DS to lead this @wphyojpl @ngachung 

@anilnatha to support answering any questions / review

**Comment by rtapella** [comment]
(Ramesh is probably going to be more helpful)

**Comment by rtapella** [comment]
@wphyojpl is this just tracking work you have elsewhere?

**Comment by wphyojpl** [comment]
This one: https://github.com/unity-sds/unity-data-services/issues/353?

**Comment by rtapella** [comment]
yes. thank you. I am going to just keep this so it's on the uiux board as well.
